package.json

```diff
@@ -69,7 +69,8 @@
     "verify-api-extract": "yarn extract-api && ./scripts/verify-extract-api.sh",
     "verify-commit": "yarn ts-node ./scripts/verify-commit.ts",
     "verify-yarn-lock": "./scripts/verify-yarn-lock.sh",
-    "workflow-results": "yarn ts-node ./scripts/cci-workflow-results.ts"
+    "workflow-results": "yarn ts-node ./scripts/cci-workflow-results.ts",
+    "postinstall": "husky install"
```

Comment:
please sort `scripts`

Reply:
Sure. Just sorted all the scripts. 

just FYI, there's a bug in VS Code, so the `-` can't be sorted correctly, but we can keep it as it is if we use a sorting tool on VS Code https://github.com/Tyriar/vscode-sort-lines/issues/11